src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/BeneficialOwnerGovernmentOrPublicAuthorityDto.java

```diff
@@ -17,6 +18,10 @@ public class BeneficialOwnerGovernmentOrPublicAuthorityDto {
     public static final String START_DATE_FIELD = "start_date";
     public static final String BENEFICIAL_OWNER_NATURE_OF_CONTROL_TYPES_FIELD = "beneficial_owner_nature_of_control_types";
     public static final String NON_LEGAL_FIRM_MEMBERS_NATURE_OF_CONTROL_TYPES_FIELD = "non_legal_firm_members_nature_of_control_types";
+    public static final String NON_LEGAL_FIRM_CONTROL_NATURE_OF_CONTROL_TYPES_FIELD = "non_legal_firm_control_nature_of_control_types";
+    public static final String TRUST_NATURE_OF_CONTROL_TYPES_FIELD = "trust_control_nature_of_control_types";
```

Comment:
the static string needs updating to TRUST_CONTROL_NATURE_OF_CONTROL_TYPES_FIELD

Reply:
As mentioned above would this have to apply to:
TRUSTEES_CONTROL_NATURE_OF_CONTROL_TYPES_FIELD ?

Reply:
ideally it should be as close as possible to the json field name

Reply:
updated